### DISCUSSION TITLE
Still can't insert into delta tables Trino 425

### DISCUSSION THREAD

**Discussion by dishkakrauch** [trinodb/trino#18830] [Q&A]
Hi everybody.

Our Trino cluster is deployed in K8S and it still throws an error like that

`SQL Error [13]: Query failed (#20230828_091632_00474_hbrtf): Writes are not enabled on the hdfs filesystem in order to avoid eventual data corruption which may be caused by concurrent data modifications on the table. Writes to the hdfs filesystem can be however enabled with the 'delta.enable-non-concurrent-writes' configuration property.`

However we'd changed configMap (I checked coordinator /etc/ files) 
![IMG_20230828_122017_186](https://github.com/trinodb/trino/assets/34490641/2dddf075-1c00-40fb-b4ed-78dd0804ba86)
we can insert into tables only with CTAS instrustion:

```
CREATE TABLE delta_adh2.stg.t_drop_me AS
(
SELECT 1 as id
FROM delta_adh2.prd.d_days
LIMIT 10
);
```

Following code will fail:

```
INSERT INTO delta_adh2.stg.t_drop_me
(id)
VALUES (2);
```

Can anybody help?

**Comment by findinpath** [comment]
The error message is misleading :( .

Trino doesn't offer support for writing on hdfs with the Delta Lake connector.

You are likely missing a few properties in your delta.properties file. 

Have a look here at this example https://github.com/trinodb/trino/blob/master/testing/trino-server-dev/etc/catalog/delta.properties

**Comment by dishkakrauch** [comment]
@findinpath I understand than Trino doesn't offer support for this connector. I've just asked for a help - may be someone solved it earlier.
Following your link I didn't find any new parameters related to HDFS and Hive except proxy but it's irrelevant.

**Comment by findinpath** [comment]
At the time of this writing, Trino doesn't offer support for writing on hdfs with the Delta Lake connector.

**Comment by dishkakrauch** [comment]
How did you test your commit https://github.com/trinodb/trino/commit/f7c64e508d7ebe8905ba65f04230af26dbac262c?
Looks like it's not discussion but like an issue.
